### PR TITLE
Many UI fixes.

### PR DESCRIPTION
**Pull request by oliverchang**
List view changes:
- Improve list view layout.
- Add "ecosystem" filter to list view.
- Sort list view by last modified time instead of ID.
- Improving vulnerability search indexing by tokenizing and including
  more fields.

Vulnerability view changes:
- Display source of vulnerabilities.
- Make long version lists scrollable.

Others:
- Override default font with Roboto.
- Tweak home page text.

### CODE REVIEW

File: gcp/appengine/frontend/src/components/List.vue [hunk 1]
```diff
@@ -83,11 +84,15 @@ export default {
   data() {
     return {
       // Set to a high limit to allow page numbers from url param to be set
-      // before results are loaded.
+      // before kesults are loaded.
```
Comment (jonathanmetzman) [inline gcp/appengine/frontend/src/components/List.vue:87]: nit: typo

File: gcp/appengine/frontend_handlers.py [hunk 2]
```diff
@@ -92,6 +94,21 @@ def add_links(bug):
       affected['fixed_link'] = _commit_to_link(repo_url, affected['fixed'])
 
 
+def add_source_info(bug, response):
+  """Add source information."""
```
Comment (jonathanmetzman) [inline gcp/appengine/frontend_handlers.py:98]: Maybe "Add source information to |response|." so that it's clearer this mutates response.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: gcp/appengine/frontend/src/components/List.vue (hunk 1 above)
Comment (oliverchang) [reply gcp/appengine/frontend/src/components/List.vue:87]: fixed!

File: gcp/appengine/frontend_handlers.py (hunk 2 above)
Comment (oliverchang) [reply gcp/appengine/frontend_handlers.py:98]: done!

**Pull request merged**